pytest_splunk_addon/cim_tests/test_templates.py

```diff
@@ -56,6 +56,13 @@ def test_cim_required_fields(
             - Check that each required field is extracted in all of the events mapped with the data model.
             - Check that if there are inter dependent fields, either all fields should be
               extracted or none of them should be extracted.
+
+        Args:
+            splunk_search_util (SearchUtil): Object that helps to search on Splunk.
+            splunk_ingest_data (fixture): Unused but required to ensure data was ingested before running test
```

Comment:
Can we remove this unused line as we are using it.

Reply:
Done